examples/test-instructions.md

```diff
@@ -0,0 +1,70 @@
+# Automating Documentation Testing with Innovation Engine
+
+## Overview
+
+This guide explains how to set up and use a GitHub Action that automates testing of your Markdown documentation using Innovation Engine. The action ensures your executable documents are accurate and robust by running tests on them. It will create issues for any errors found and prevent pull requests from merging until those errors are resolved.
```

Comment:
How does this action prevent pull requests from merging?

Reply:
technically, other settings will need to be configured for the prevention step but it does show errors in docs as part of the PR. Now, someone could theoretically still merge the PR. updated the verbiage as a result